docs/quick-install.md

```diff
@@ -1,175 +1,184 @@
 ```
 
-__Windows:__ Ensure that the following dependencies are installed:
+***
 
-[.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+##### Option 2: From the documentation site 
+If you are reading this content from the SignalFx documentation site, then SignalFx recommends that you access the Integrations page in the SignalFx UI to copy the pre-populated installation code.  
 
-[Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+1. Log in to SignalFx and click the **Integrations** tab to open the Integrations page. Look for the SignalFx Smart Agent tile. You can search for it by name, or find it in the **Essential Services** section.
+2. Under **Essential Services**, click **SignalFx Smart Agent**.
+3. Click **Setup**.
+4. Locate the text box for Windows users.
+5. Copy, paste, and run the code in your command line. (The code within the tile is already populated with your realm and your organization's access token.)  
 
-* Get your API\_TOKEN from:  __Organization Settings => Access Token__ tab in the SignalFx application.
+The agent will be installed as a Windows service and will log to the Windows Event Log.
+</p>
+</details>
 
-* Determine YOUR\_SIGNAL\_FX_REALM from: your profile page in the SignalFx web application.
+***
 
-To install the Smart Agent on a single Windows host, enter:
+### Step 2. Confirm your installation
 
-```sh
-& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"};; 
-ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
-```
+1. From the command line, enter the following command to confirm that your Smart Agent is functional:
 
-The agent will be installed as a Windows service and will log to the Windows Event Log.
+    ```sh
+    sudo signalfx-agent status
+    ```
 
+    The return should be similar to the following example:  
 
-### Step 2. Confirm your Installation 
+    ```sh
+    SignalFx Agent version:           4.7.6
+    Agent uptime:                     8m44s
+    Observers active:                 host
+    Active Monitors:                  16
+    Configured Monitors:              33
+    Discovered Endpoint Count:        6
+    Bad Monitor Config:               None
+    Global Dimensions:                {host: my-host-1}
+    Datapoints sent (last minute):    1614
+    Events Sent (last minute):        0
+    Trace Spans Sent (last minute):   0
+    ```
 
-To confirm the SignalFx Smart Agent installation is functional on either platform, enter:
+2. Enter the following commands to verify the installation:
```

Comment:
How is "confirm that your Smart Agent is functional" in step 1 different from "verify the installation" in step 2?

Reply:
I guess they are both trying to achieve the same purpose, which is to validate / confirm / verify that the installation was successful. 

I updated both steps to say, "To confirm your installation, enter the following command on the Linux or Windows command line:..."